src/swagger-specs/sandbox.yaml

```diff
@@ -614,12 +662,15 @@ definitions:
         example: 180e1210-d33a-11e9-8b17-d9e538719ued
       name:
         type: string
+        description: The identifier that will be used to access the sandbox in future requests. This value must be unique, and best practice is to make it as descriptive as possible. This value cannot contain any spaces or special characters.
         example: mysandbox
       title:
         type: string
+        description: A human-readable name used for display purposes in the Platform user interface.
         example: My Sandbox
       state:
         type: string
+        description: Sandboxes take roughly 30 seconds to be provisioned by the system, after which their state will become “active” or “failed”.
```

Comment:
we should backtick these values `active` `failed`
I would also reframe this slightly. It is: "The state of the sandbox and has two values, `active` `failed`. A sandbox becomes `active` once it has been successfully provisioned." or sth to that effect